yigitcanTask/force-app/main/default/classes/searchDetailsController.cls

```diff
@@ -0,0 +1,13 @@
+public class searchDetailsController {
```

Comment:
why do you need this class?
if it is not used - just delete it and do not include in pull request